src/main/java/de/samply/share/client/model/centralsearch/UploadStats.java

```diff
@@ -23,4 +24,9 @@ public UploadStats(String lastUploadTimestamp) {
   public String getLastUploadTimestamp() {
     return lastUploadTimestamp;
   }
+
+  public void setLastUploadTimestamp(String lastUploadTimestamp) {
```

Comment:
Please create a Test for the serialisation/deserialization and look if this otherwise not used setter can be removed.